Sources/Commands/SwiftTool.swift

```diff
@@ -339,27 +345,20 @@ public class SwiftTool<Options: ToolOptions> {
         }
     }
 
+    private lazy var _destination: Result<Destination, AnyError> = {
```

Comment:
this is used to create the destination for the toolchain but will move that in there